assets/components/Links.vue

```diff
@@ -14,10 +14,15 @@
 
     <dropdown class="dropdown-end" v-if="config.user">
       <template #trigger>
-        <img
-          class="ring-base-content/60 size-6 max-w-none rounded-full p-px ring-1"
-          :src="withBase('/api/profile/avatar')"
-        />
+        <template v-if="disableAvatars">
```

Comment:
As I wrote in the original issue, I feel like this should just happen automatically if email is blank. Are you still providing an email address? Most people don't read docs so it would be good to have better defaults. Thoughts?